.github/workflows/pullrequestclose.yml

```diff
@@ -0,0 +1,28 @@
+name: Pull Request Close
```

Comment:
[note] https://github.com/viamrobotics/rdk/blob/main/.github/workflows/pullrequestclose.yml

Reply:
just to be clear, all instances of `qaworkflow` will have to be changed to main for this file, right?

Reply:
mind please posting the diff?